### DISCUSSION TITLE
Why is the documentation remarking the possiblity of using useTranslations  in async components if we have getTranslations?

### DISCUSSION THREAD

**Discussion by onarbf** [amannn/next-intl#610] [General]
Hi! I've been working around a template for Nextjs13 + Sanity, and it was the first time with this package. Really blow my mind how useful it is nowadays where  all the pipeline we have defined is being re-built because the SSR revolution.

I was trapped trying to use the unstable_setRequestLocale(locale), but it has been impossible.

Then I read that exists the getTranslations. In the docs, it only appears in the Generate Metadata part. I don't understand why... is there any side effect in the cache or something like that? Why are you not speaking about this part of the package, when is the best solution for the async problem? Thank you man. Btw, you can check my repo here:

https://github.com/onarbf/nextity

Anyway, thanks for this tool. Docs in Nextjs about this are not clear at all yet.

**Comment by amannn** [comment]
Just this week [I've streamlined `getTranslations`](https://github.com/amannn/next-intl/pull/600) for easier usage in components and updated the docs of the release candidate accordingly: [Using internationalization in Server Components](https://next-intl-docs-git-feat-next-13-rsc-next-intl.vercel.app/docs/environments/server-client-components#using-internationalization-in-server-components).

I hope this helps!
